### PR TITLE
Modal: Create Event

### LINKED ISSUE #132: Frontend: Modal View to create a new events
When I am logged in as an admin, I want to be able to create a new event and fill in the info for it.

Create a Modal View to create a new Event like shown below:

image

This component should use the API developed here #133 

Note: Create the components with the appropriate style. Please check the wireframes https://www.figma.com/file/9gN79UmpDPrAZk5jY4V7Ww/Study-Group-Management-System?node-id=0%3A1

### PR DESCRIPTION
**Pull request by Deepti-Dk**
# Description

This PR is for create-event modal.

Fixes #132 

# How to test?

tested in swagger

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by MaherHussain — APPROVED** [review]
yessss

File: src/client/containers/ModalCreateEvents/ModalCreateEvents.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Button } from '../../components/Button/Button';
+
+import './ModalCreateEvents.styles.css';
+import DropDown from '../../components/Dropdown/DropDown';
+import DateTime from '../../components/DateTime/DateTime';
+import Modal from '../../components/Modal/Modal';
+
+const companies = [
```
Comment (tpetrina) [inline src/client/containers/ModalCreateEvents/ModalCreateEvents.js:9]: You should push `companies` and `eventTypes` through props and use this hard-coded list in storybook.

File: src/client/containers/ModalCreateEvents/ModalCreateEvents.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  {
+    id: '01',
```
Comment (tpetrina) [inline src/client/containers/ModalCreateEvents/ModalCreateEvents.js:11]: Id should be number, not a string, right?

File: src/client/containers/ModalCreateEvents/ModalCreateEvents.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            value={eventtype}
+            setValue={setEventType}
+            items={eventTypes}
+          />
+        </div>
+        <div className="create-event-datetime">
+          <DateTime value={datetime} setValue={setDateTime} />
+        </div>
+        <div className="create-event-registration-button">
+          <Button>Submit</Button>
```
Comment (tpetrina) [inline src/client/containers/ModalCreateEvents/ModalCreateEvents.js:65]: Handle onClick in this modal and call the `props.onCreate`.

File: src/client/containers/ModalCreateEvents/ModalCreateEvents.styles.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.create-event-modal {
+  margin: auto;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
```
Comment (tpetrina) [inline src/client/containers/ModalCreateEvents/ModalCreateEvents.styles.css:7]: Is this really needed here?

File: src/client/containers/ModalCreateEvents/ModalCreateEvents.stories.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+export default { title: 'Modal to create events' };
+
+export const Events = () => {
+  return (
+    <ModalCreateEvents
+      events={events}
+      companies={companies}
+      onSubmit={(data) => {
+        console.log(data);
```
Comment (MaherHussain) [inline src/client/containers/ModalCreateEvents/ModalCreateEvents.stories.js:43]: remove console.log here

Comment (Deepti-Dk) [reply src/client/containers/ModalCreateEvents/ModalCreateEvents.stories.js]: done

**Review by tpetrina — APPROVED** [review]
Good job. Please remove that warning.

**Review by MaherHussain — APPROVED** [review]

**Pull request merged**